src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1201,13 +1201,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * Post /courses/:courseId/students/:studentLogin : Add the given user to the students of the course so that the student can access the course
+     * POST courses/:courseId/students/:studentLogin : Add the given user to the students of the course so that the student can access the course
```

Comment:
```suggestion
     * Post /courses/:courseId/students/:studentLogin : Add the given user to the students of the course so that the student can access the course
```